### PR TITLE
rewrite the beginning of xstore blogpost

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

File: _posts/2023-08-10-xstore.md [hunk 1]
```diff
@@ -1,69 +1,52 @@
...
-an appstore after the example
-of the popular Open Source [F-Droid store](https://f-droid.org/).
-
-## How can I download apps from xstore?
 
-You can **scan this QR code with Delta Chat**
-([*do you need a test account?*](#tryout))
-to message [xstore@testrun.org](mailto:xstore@testrun.org)
-and try out the store:
+It doesn't need to be hard to write, distribute or find end-to-end encrypted apps anymore.
```
Comment (missytake) [inline _posts/2023-08-10-xstore.md]: Hm, not a fan. This speaks mostly to developers; as they are not the main intended audience of the blogpost, it shouldn't be the first sentence.

On the other hand, I'm not sure how many people know what a "super app"/"everything-app" is, so the current beginning doesn't fit this criteria either...

File: _posts/2023-08-10-xstore.md [hunk 2]
```diff
@@ -1,69 +1,52 @@
...
-With one tap you can download them,
-and with another tap,
-share them to any chat.
-If you want to use it on your own,
-just share it to your "Saved Messages" chat.
+<img src="../assets/blog/gordon_superapp.jpg" width="340" style="float:right; margin-left:1em;" />  
+Others are hand-wavingly talking about so called "super-app" or "everything app" architectures, 
+but we are delivering it,
+decentralized and end-to-end encrypted
+and open to collaborators from other messaging efforts.
```
Comment (missytake) [inline _posts/2023-08-10-xstore.md]: Maybe we could move this paragraph to the top.

File: _posts/2023-08-10-xstore.md [hunk 3]
```diff
@@ -1,69 +1,52 @@
...
-<img src="../assets/blog/gordon_superapp.jpg" width="340" style="float:right; margin-left:1em;" />  
-As [Gordon Brander](https://mastodon.social/@gordon) already stated -
-with [Private Apps](help#webxdc),
-Delta Chat is more than a messenger.
-You can share Private Apps with any other Delta Chat user
-and use them collaboratively;
-for example to play chess
-or to edit your household's shopping list.
-
 <!-- toot: make fun of Musk's X-thing: while some billionaires are talking about everything-apps, we are delivering it: [xstore@testrun.org](mailto:xstore@testrun.org) -->
```
Comment (missytake) [inline _posts/2023-08-10-xstore.md]: If the first paragraph is a comment, the description on https://staging.delta.chat/711/en/blog is empty.

**Comment by hpk42** [comment]
On Wed, Aug 09, 2023 at 11:48 -0700, missytake wrote:

> @missytake commented on this pull request.
> -You can **scan this QR code with Delta Chat**
> -([*do you need a test account?*](#tryout))
> -to message ***@***.******@***.***)
> -and try out the store:
> +It doesn't need to be hard to write, distribute or find end-to-end encrypted apps anymore. 
> 
> Hm, not a fan. This speaks mostly to developers; as they are not the main intended audience of the blogpost, it shouldn't be the first sentence.

I'd like to address developers with the blog post but i get your point anyway.

File: _posts/2023-08-10-xstore.md (hunk 1 above)
Comment (hpk42) [reply _posts/2023-08-10-xstore.md]: changed it.

**Pull request merged**